[2024-03-11T16:08:15.496+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Lead_Scoring_Data_Engineering_Pipeline.checking_raw_data_schema scheduled__2024-03-10T00:00:00+00:00 [queued]>
[2024-03-11T16:08:15.500+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Lead_Scoring_Data_Engineering_Pipeline.checking_raw_data_schema scheduled__2024-03-10T00:00:00+00:00 [queued]>
[2024-03-11T16:08:15.501+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-03-11T16:08:15.520+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): checking_raw_data_schema> on 2024-03-10 00:00:00+00:00
[2024-03-11T16:08:15.524+0000] {standard_task_runner.py:60} INFO - Started process 399 to run task
[2024-03-11T16:08:15.529+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'Lead_Scoring_Data_Engineering_Pipeline', 'checking_raw_data_schema', 'scheduled__2024-03-10T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/Lead_scoring_data_pipeline/lead_scoring_data_pipeline.py', '--cfg-path', '/tmp/tmpmdpdb4n5']
[2024-03-11T16:08:15.529+0000] {standard_task_runner.py:88} INFO - Job 6: Subtask checking_raw_data_schema
[2024-03-11T16:08:15.565+0000] {task_command.py:423} INFO - Running <TaskInstance: Lead_Scoring_Data_Engineering_Pipeline.checking_raw_data_schema scheduled__2024-03-10T00:00:00+00:00 [running]> on host 19276f21f48e
[2024-03-11T16:08:15.626+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Lead_Scoring_Data_Engineering_Pipeline' AIRFLOW_CTX_TASK_ID='checking_raw_data_schema' AIRFLOW_CTX_EXECUTION_DATE='2024-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-10T00:00:00+00:00'
[2024-03-11T16:08:15.627+0000] {logging_mixin.py:188} INFO - Exception thrown in raw_data_schema_check : [Errno 2] No such file or directory: '/airflow/dags/Lead_scoring_data_pipeline/data/leadscoring.csv'
[2024-03-11T16:08:15.627+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-11T16:08:15.630+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=Lead_Scoring_Data_Engineering_Pipeline, task_id=checking_raw_data_schema, execution_date=20240310T000000, start_date=20240311T160815, end_date=20240311T160815
[2024-03-11T16:08:15.658+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-11T16:08:15.674+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
